react/components/molecules/key-values/key-values.js

```diff
@@ -1,37 +1,101 @@
 import React, { PureComponent } from "react";
-import { View, ViewPropTypes } from "react-native";
+import { StyleSheet, View, ViewPropTypes } from "react-native";
 import PropTypes from "prop-types";
 
 import { KeyValue } from "../key-value";
+import { isTabletSize } from "../../../util";
 
 export class KeyValues extends PureComponent {
     static get propTypes() {
         return {
             items: PropTypes.arrayOf(
                 PropTypes.shape({
                     key: PropTypes.string.isRequired,
-                    value: PropTypes.string.isRequired
+                    value: PropTypes.string.isRequired,
+                    keyColor: PropTypes.string,
+                    valueColor: PropTypes.string,
+                    icon: PropTypes.string,
+                    iconBackgroundColor: PropTypes.string,
+                    iconColor: PropTypes.string,
+                    iconSize: PropTypes.number,
+                    iconHeight: PropTypes.number,
+                    iconWidth: PropTypes.number,
+                    iconStrokeWidth: PropTypes.number,
+                    pressable: PropTypes.bool,
+                    onPress: PropTypes.func,
+                    onButtonIconPress: PropTypes.func,
+                    onLongPress: PropTypes.func
                 })
             ).isRequired,
+            twoColumns: PropTypes.bool,
             style: ViewPropTypes.style
         };
     }
 
     static get defaultProps() {
-        return { style: {} };
+        return {
+            twoColumns: isTabletSize(),
+            style: {}
+        };
     }
 
+    _style = () => {
+        return [this.props.twoColumns ? styles.keyValuesColumns : {}, this.props.style];
+    };
+
+    _keyValueWrapperStyle = () => {
+        return this.props.twoColumns ? styles.twoColumns : {};
+    };
+
+    _keyValueStyle = index => {
+        return this.props.twoColumns && index % 2 > 0 ? styles.keyValueTwoColumns : {};

```

Comment:
`keyValueTwoColumns` seems misleading because the name indicates that it should be applied to all key value when there are two columns, but the truth is it's only applied to the right column. Please rename accordingly.

Reply:
Suggestion: `keyValueColumnRight`